packages/pn-personafisica-webapp/src/pages/NuovaDelega.page.tsx

```diff
@@ -106,10 +106,7 @@ const NuovaDelega = () => {
     cognome: '',
     selectTuttiEntiOrSelezionati: 'tuttiGliEnti',
     expirationDate: tomorrow,
-    enteSelect: {
-      name: '',
-      uniqueIdentifier: '',
-    },
+    enteSelect: [],
```

Comment:
@NicolaGiornetta @leleOFA change this attribute to enti

Reply:
do it